docker-compose.yaml

```diff
@@ -1,26 +1,26 @@
-version: '3.9.1'
+version: "3.9.1"
 
 # You must manually run `ankerctl config import` atleast once
 # docker run \
 # -v ankerctl_vol:/root/.config/ankerctl \
 # -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
 # ankerctl config import /tmp/login.json
 services:
-  ankerctl:
-    image: ankerctl/ankerctl:latest
-    container_name: ankerctl
-    restart: unless-stopped
-    build: .
-    environment:
-      - FLASK_PORT=4470
-      - FLASK_HOST=0.0.0.0
-    volumes:
-      - ankerctl_vol:/root/.config/ankerctl
-    ports:
-      - 127.0.0.1:4470:4470
-    entrypoint: "/app/ankerctl.py"
-    command: ["webserver", "run"]
+    ankerctl:
```

Comment:
Changing the line indentations from 2 spaces to 4 spaces makes it kind of hard to see what changed.

it looks like working_dir was added?

could we keep the indenting the same and handle a linting refactor in a separate PR? (Ideally automated to accommodate different editor defaults)

Reply:
See #72